test/e2e/metamask-ui.spec.js

```diff
@@ -515,136 +515,6 @@ describe('MetaMask', function () {
     });
   });
 
```

Comment:
Do we already have e2e for Token Allowance Flow, otherwise it will be nicer to add new e2e as the tests below are removed.